modules/3d/src/rgbd/volume_impl.cpp

```diff
@@ -9,17 +9,317 @@
 #include "hash_tsdf_functions.hpp"
 #include "color_tsdf_functions.hpp"
 #include "opencv2/imgproc.hpp"
+#include "opencv2/3d.hpp"
 
 namespace cv
 {
 
+
```

Comment:
Please provide a link to the author's page.
AFAIK they are somewhere in the header file in `rgbd` module which contains these tables.